MesnardBarba-CiSE.md

```diff
@@ -271,14 +272,14 @@ Thus, we adapted "old" cuIBM to be compatible with the oldest version of _Cusp_
 The case at angle-of-attack 35 degrees and Reynolds number 2000 now gave an appreciable difference compared with our previous study: 
 the instantaneous force coefficients start to slowly drop after about 60 time units (Figure 11(c)). 
 Now, this is _really_ the same code, with only a difference in the _version_ of the linear algebra library. 
-Repeating the case with the most current version of cuIBM and the same version of _Cusp_ (0.4.0) leads to the same force signals, with a slight drop towards the end (Figure 11(d)). 
+Repeating the case with the most current version of cuIBM and the same version of _Cusp_ (0.4.0) leads to the same force (the same as which?) signals, with a slight drop towards the end (Figure 11(d)). 
```

Comment:
It says in the parenthesis (0.4.0) so reader can compare that it matches the previous statement.